docs/setup_installation/aws/getting_started.md

```diff
@@ -226,87 +231,92 @@ Paste the *Access Key ID* and the *Secret Access Key* into [managed.hopsworks.ai
 
-Proceed to the [S3 Management Console](https://s3.console.aws.amazon.com/s3/home) and click on *Create bucket*:
-<p align="center">
-  <figure>
-    <a  href="../../../assets/images/setup_installation/managed/aws/create-s3-bucket-1.png">
-      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/aws/create-s3-bucket-1.png" alt="Create an S3 bucket">
-    </a>
-    <figcaption>Create an S3 bucket</figcaption>
-  </figure>
-</p>
+The Hopsworks clusters deployed by [managed.hopsworks.ai](https://managed.hopsworks.ai) store their data in an S3 bucket in your AWS account.
 
-Name your bucket and select the region where your Hopsworks cluster will run. Click on *Create bucket* at the bottom of the page.
 
-<p align="center">
-  <figure>
-    <a  href="../../../assets/images/setup_installation/managed/aws/create-s3-bucket-2.png">
-      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/aws/create-s3-bucket-2.png" alt="Create an S3 bucket">
-    </a>
-    <figcaption>Create an S3 bucket</figcaption>
-  </figure>
-</p>
+To create the bucket run the following command, replacing *BUCKET_NAME* with the name you want for your bucket and setting the region to the aws region in which you want to run your cluster.
 
-## Step 3: Creating Instance profile
+!!! warning
+    The bucket must be in the same region as the hopsworks cluster you are going to run
 
-!!! note 
-    If you prefer using terraform, you can skip this step and the remaining steps, and instead follow [this guide](../common/terraform.md#getting-started-with-aws).
+```bash
+aws s3 mb s3://BUCKET_NAME --region us-east-2
+```
 
-Hopsworks cluster nodes need access to certain resources such as S3 bucket and CloudWatch.
 
-Follow the instructions in this guide to create an IAM instance profile with access to your S3 bucket: [Guide](https://docs.aws.amazon.com/codedeploy/latest/userguide/getting-started-create-iam-instance-profile.html#getting-started-create-iam-instance-profile-console)
+## Step 3: Creating Instance profile
 
-When creating the policy, paste the following in the JSON tab.
+Hopsworks cluster nodes need access to certain resources such as the S3 bucket you created above, an ecr repository, and CloudWatch.
+
+
+First, create an instance profile by running:
+```bash
+aws iam create-instance-profile --instance-profile-name hopsworksai-instances
+```
+
+We will now create a role with the needed permissions for this instance profile. 
+Start by creating a file named *assume-role-policy.json* containing the following:
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "ec2.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+
+Run the following to create the role:
+
+```bash
+aws iam create-role perm --role-name hopsworksai-instances \
```

Comment:
It doesn't seem to work with aws cli v1.x, can we explicitly state that on the documentation
```Note: AWS CLI version 2, the latest major version of the AWS CLI, is now stable and recommended for general use. For more information, see the AWS CLI version 2 installation instructions at: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html

usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help

Unknown options: perm```

Reply:
This is a typo the perm should not be there. I'll remove it